app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,8 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
```

Comment:
The default value of this property is "true". Is there a reason to explicitly put it on application tag?

Reply:
Yes, there is a reason for that. 
In Andriod P if we don't explicit this property an exception will occur.

"CLEARTEXT communication to 142.93.124.175 not permitted by network security policy"

Reply:
Thanks for clarifying!